auctions/README.md

```diff
@@ -0,0 +1,7 @@
+# Overview
```

Comment:
Can you add more information in general regarding the entire auction project? This is just 1 contract within it so an overview of the project in general is good